pkg/volume/csi/csi_plugin.go

```diff
@@ -31,26 +29,6 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
-const (
-	csiPluginName              = "kubernetes.io/csi"
-	csiVolAttribsAnnotationKey = "csi.volume.kubernetes.io/volume-attributes"
```

Comment:
I think it is better to leave `csiPluginName` and `csiVolAttribsAnnotationKey` where they are

Reply:
Thanks for the review, but may I know the reason?

Reply:
Other Volume plugins do like this. Feel free to ignore this comment.